crawler/src/main/java/com/movie/recommender/crawler/Films.java

```diff
@@ -0,0 +1,85 @@
+package com.movie.recommender.crawler;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+import java.io.IOException;
+
+public class Films {
```

Comment:
rename the class to `TheMovieDbClient` to clarify its purpose as the client for interacting with The Movie Database API